cdap-app-fabric/src/main/java/co/cask/cdap/config/ConfigStore.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.config;
+
+import java.util.List;
+
+/**
+ * Configuration Store.
+ */
+public interface ConfigStore {
+
+  /**
+   * Create a Configuration.
+   * @param namespace Namespace.
```

Comment:
Parameters do not end with a period nor start with a capital letter:
```
@param namespace the configuration namespace
@param type the configuration type
@param config the configuration object
@throws Exception
```

See http://www.liferay.com/community/wiki/-/wiki/Main/Javadoc+Guidelines#section-Javadoc+Guidelines-@param+tags